helpers/hooks-list.json

```diff
@@ -156,6 +156,10 @@
       "name": "useOutsideClickRef",
       "description": "A hook that can track a click event outside a ref. Returns a callbackRef."
     },
+    {
+      "name": "usePreventBodyScroll",
+      "description": "Prevent Body Scroll"
```

Comment:
Let's change the description everywhere to 'A react hook to prevent body scroll. Optionally takes a condition `when` which disables body scroll only when the condition is true. '